app/config/config.neon

```diff
@@ -1,4 +1,5 @@
 includes:
+	- database.neon
```

Comment:
It's best practise to use `parameters` section to store credentials, not particular database extension.

These can be used on more places or dblayers (e.g. for Doctrine) with only 1 place to modify.